### PR TITLE
add various distros

### PR DESCRIPTION
**Pull request by flokli**

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -283,6 +314,12 @@ targets:
       - netflix.com
       - www.netflix.com
     twitter: "@Netflixhelps"
+  NixOS:
+    href: https://nixos.org
+    hosts:
+      - nixos.org
```
Comment (mweinelt) [inline conf.yaml]: cache.nixos.org is very relevant here

File: conf.yaml [hunk 2]
```diff
@@ -146,6 +171,12 @@ targets:
       - de.foursquare.com
     icon: "fa:foursquare"
     twitter: "@Foursquare"
+  Gentoo:
+    href: https://www.gentoo.org
+    hosts:
+      - www.gentoo.org
```
Comment (mweinelt) [inline conf.yaml]: maybe `distfiles.gentoo.org`?

File: conf.yaml [hunk 3]
```diff
@@ -69,11 +75,23 @@ targets:
...
+    twitter: "@coreos"
   Coveralls:
     href: https://coveralls.io/
     hosts:
       - coveralls.io
     twitter: "@CoverallsApp"
+  Debian:
+    href: https://www.debian.org
+    hosts:
+      - www.debian.org
```
Comment (mweinelt) [inline conf.yaml]: maybe `security.debian.org`

File: conf.yaml [hunk 4]
```diff
@@ -48,6 +48,12 @@ targets:
       - itunes.apple.com
     icon: "fa:apple"
     twitter: "@apple"
+  Archlinux:
+    href: https://www.archlinux.org
+    hosts:
+      - www.archlinux.org
```
Comment (mweinelt) [inline conf.yaml]: maybe `aur.archlinux.org`

**Comment by flokli** [comment]
updated.

**Pull request merged**